totalsize/total.py

```diff
@@ -139,9 +149,20 @@ def _readable_amount(self, amount, byte=False):
 
 
 class Playlist:
-    def __init__(self, url, format_sel, retries=0):
+    def __init__(self, url, format_sel, retries=0, cookies=None):
+        YTDL_OPTS_LOCAL = YTDL_OPTS
+        if cookies:
+            if not os.path.isfile(cookies):
+                raise CookieFileError  
+            try:
+                cj = http.cookiejar.MozillaCookieJar()
+                cj.load(cookies, ignore_discard=True, ignore_expires=True)
+            except:
```

Comment:
Catch-all except statements are not cool. Please investigate exactly what exception(s) are thrown when the cookie file is invalid, and then catch only those.

Reply:
Fixed, is this correct?